### PR TITLE
Update and rename wirtschaftsuniversitat-wien-master-management.csl t…

### PR DESCRIPTION
**Pull request by georgd**
…o wirtschaftsuniversitat-wien-apa-based.csl

### CODE REVIEW

File: wirtschaftsuniversitat-wien-apa-based.csl [hunk 1]
```diff
@@ -2,20 +2,20 @@
 <style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" name-delimiter="/" initialize="false" initialize-with="." demote-non-dropping-particle="never">
   <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
   <info>
-    <title>Wirtschaftsuniversität Wien - Master Management</title>
-    <id>http://www.zotero.org/styles/wirtschaftsuniversitat-wien-master-management</id>
-    <link href="http://www.zotero.org/styles/wirtschaftsuniversitat-wien-master-management" rel="self"/>
+    <title>Wirtschaftsuniversität Wien - APA Based (English and German)</title>
```
Comment (rmzelle) [inline wirtschaftsuniversitat-wien-apa-based.csl]: @georgd, would `<title>Wirtschaftsuniversität Wien - APA</title>` be acceptable? We never include languages in the title for multi-lingual styles, and "Based" is a bit redundant.

**Comment by georgd** [comment]
OK, we can drop the languages.

With the "Based" part I wanted to indicate that it's a quite customized APA. Authors formats don't look like APA at all. At WU we have other institutes that use APA variants that are by far closer to the original. Do you have a better idea?

**Comment by georgd** [comment]
OK, we can drop the languages.

With the "Based" part I wanted to indicate that it's a quite customized APA. Authors formats don't look like APA at all. At WU we have other institutes that use APA variants that are by far closer to the original. Do you have a better idea?

**Comment by rmzelle** [comment]
> With the "Based" part I wanted to indicate that it's a quite customized APA. Authors formats don't look like APA at all. At WU we have other institutes that use APA variants that are by far closer to the original. Do you have a better idea?

If "APA" is not a good descriptor it might be better to stick with the department naming.

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**